src/core/flow.py

```
@@ -0,0 +1,74 @@
+        tokens = self.asr(input_batch, self.sr)
+
+        for i, task in enumerate(tasks):
+            task_tokens = tokens[chunk_task_map == i]
+            task.tokens = task_tokens
+
+
+class AudioPostProcessorHandler(BaseTaskHandler):
+    def __init__(self, processor: AudioProcessor):
+        self.processor = processor
+
+    def handle(self, *tasks: AudioTask):
```

Ruff (0.12.2)

Missing return type annotation for public function `handle`

Add return type annotation: `None`

(ANN201)